### PR TITLE
Add SQL parser test case for ResetScalingStatement (#15733)

### LINKED ISSUE #15733: [OpenForce 2022]  Add SQL parser test case for `ResetScalingStatement`
Hi community,

This issue is for [OpenForce 2022](https://github.com/apache/shardingsphere/discussions/12683#discussioncomment-2139441)

### Background

`RESET SCALING jobId` is a syntax in [DistSQL RAL](https://shardingsphere.apache.org/document/current/en/user-manual/shardingsphere-proxy/distsql/syntax/ral/), when `ShardingSphereSQLParserEngine` receives this SQL text, it parses it as `ResetScalingStatement`. Now we need to add a test case for this parsing process to assert that its parsing is correct.

### Aim

Add SQL parser test case for `ResetScalingStatement`.

- Affected files:

  - `SQLParserTestCases` (modify)

  - `shardingsphere-test/shardingsphere-parser-test/src/main/resources/case/ral/update.xml` (modify)

  - `shardingsphere-test/shardingsphere-parser-test/src/main/resources/sql/supported/ral/update.xml` (modify)

  - `shardingsphere-test/shardingsphere-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/jaxb/cases/domain/statement/distsql/ral/scaling/ResetScalingStatementTestCase.java` (create)

  - `shardingsphere-test/shardingsphere-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/asserts/statement/distsql/ral/impl/scaling/update/ResetScalingStatementAssert.java` (create)

  - `UpdatableScalingRALStatementAssert` (modify)

- After all, execute `DistSQLParserParameterizedTest#assertDistSQL()` for validation

### Basic Qualifications

- Java
- Maven

### Example FYI

- `ApplyScalingStatement`
- `ApplyScalingStatementTestCase`
- `ApplyScalingStatementAssert`
- search for "apply-scaling"

### PR DESCRIPTION
**Pull request by pparthenhs**
Fixes #15733.

### CODE REVIEW

File: shardingsphere-test/shardingsphere-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/jaxb/cases/domain/SQLParserTestCases.java [hunk 1]
```diff
@@ -197,11 +197,7 @@
...
 import org.apache.shardingsphere.test.sql.parser.parameterized.jaxb.cases.domain.statement.distsql.ral.ShowVariableStatementTestCase;
 import org.apache.shardingsphere.test.sql.parser.parameterized.jaxb.cases.domain.statement.distsql.ral.UnlabelInstanceStatementTestCase;
-import org.apache.shardingsphere.test.sql.parser.parameterized.jaxb.cases.domain.statement.distsql.ral.scaling.ApplyScalingStatementTestCase;
-import org.apache.shardingsphere.test.sql.parser.parameterized.jaxb.cases.domain.statement.distsql.ral.scaling.CheckScalingStatementTestCase;
-import org.apache.shardingsphere.test.sql.parser.parameterized.jaxb.cases.domain.statement.distsql.ral.scaling.RestoreScalingSourceWritingStatementTestCase;
-import org.apache.shardingsphere.test.sql.parser.parameterized.jaxb.cases.domain.statement.distsql.ral.scaling.ShowScalingCheckAlgorithmsStatementTestCase;
-import org.apache.shardingsphere.test.sql.parser.parameterized.jaxb.cases.domain.statement.distsql.ral.scaling.StopScalingSourceWritingStatementTestCase;
+import org.apache.shardingsphere.test.sql.parser.parameterized.jaxb.cases.domain.statement.distsql.ral.scaling.*;
```
Comment (RaigorJiang) [inline shardingsphere-test/shardingsphere-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/jaxb/cases/domain/SQLParserTestCases.java:200]: Please do not use * to import dependencies

**Comment by RaigorJiang** [comment]
Hi @pparthenhs 
You should run `./mvnw clean install` `and make sure the local compilation passes.

**Comment by RaigorJiang** [comment]
@pparthenhs 
Very concise and accurate PR, thanks for your work.
I'll merge it after the CI is successful.

**Comment by pparthenhs** [comment]
> @pparthenhs Very concise and accurate PR, thanks for your work. I'll merge it after the CI is successful.

I am happy to contribute !!

**Review by RaigorJiang — APPROVED** [review]

**Pull request merged**